Makefile

```diff
@@ -1,9 +1,9 @@
 DESTDIR =
 PREFIX  = /usr/local
 
-BASHDIR = /etc/bash_completion.d
+BASHDIR = $(PREFIX)/etc/bash_completion.d
 ZSHDIR  = /usr/share/zsh/vendor-completions
```

Comment:
Oh I totally forgot about `pkg-config`! I just tried it on macOS and Linux, which were under `/usr/local` and `/usr/share/local`. Well that’s proof enough that the directories aren’t consistent. That’s something I’d really like to get set up.

exa uses the `bash_completion` package, rather than `bash`, which doesn’t provide enough functionality out-of-the-box. Its FAQ [actually recommends using `pkg-config`](https://github.com/scop/bash-completion#faq) as well, but its directory does default to `/etc`, even though Homebrew (among others, probably) will [move it somewhere else](https://github.com/Homebrew/homebrew-core/blob/master/Formula/bash-completion.rb), probably for the reasons you mention.

I can’t find anything for zsh so it’s looking more and more likely we should just leave it un-prefixed.